.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I believe this becomes the default template for *all* issues tho. Even if a user creates an issue.

Reply:
Yes, it seems to (I saw that in pantry core). I'll switch to a named non-default template if possible.